azavea/temperate
# Feature/kak/accordion graph containers#15

## Overview

Adds collapsible Bootstrap accordion section for 'all indicators' to the Indicators tab, with some dummy text.

### Demo

Collapsed:
![image](https://user-images.githubusercontent.com/960264/32075854-05a0ecd0-ba6c-11e7-9889-b0e9409e051b.png)

Top section expanded:
![image](https://user-images.githubusercontent.com/960264/32075870-13eae9ee-ba6c-11e7-8db1-b7d787b1383f.png)


### Notes

This section is in the Marvel prototype [here](https://marvelapp.com/29h65ff/screen/32528118) but not in the functional prototype HTML in this repository.

Had to change some of the custom `card` class styling to fix card collapse; the `min-height` on the card was causing the content to show/hide on collapse toggle, but the card would remain visible at full height.

[Bootstrap collapse docs](https://getbootstrap.com/docs/4.0/components/collapse/)
[ngx-bootstrap collapse docs](https://valor-software.com/ngx-bootstrap/#/collapse)

## Testing Instructions

 * `./scripts/update`
 * `./scripts/server --angular`
 * visit http://localhost:4210/indicators
 * collapsible accordion sections should behave as expected

Closes #15
